lib/util/cache/repository/impl/s3.ts

```diff
@@ -54,14 +56,22 @@ export class RepoCacheS3 extends RepoCacheBase {
 
   async write(data: RepoCacheRecord): Promise<void> {
     const cacheFileName = this.getCacheFileName();
+    const stringifiedCache = JSON.stringify(data);
     const s3Params: PutObjectCommandInput = {
       Bucket: this.bucket,
       Key: cacheFileName,
-      Body: JSON.stringify(data),
+      Body: stringifiedCache,
       ContentType: 'application/json',
     };
     try {
       await this.s3Client.send(new PutObjectCommand(s3Params));
+      if (process.env.RENOVATE_X_REPO_CACHE_FORCE_LOCAL === 'true') {
```

Comment:
```suggestion
      if (is.nonEmptyString(process.env.RENOVATE_X_REPO_CACHE_FORCE_LOCAL)) {
```
This is currently not doing what is described in the docs. 

It expects 'true' rather checking for any setting. 
We should change the docs or how this is detected.